src/frontend/src/binder/query.rs

```diff
@@ -187,6 +197,20 @@ impl Binder {
         Ok(FieldOrder { index, direct })
     }
 
+    /// `ORDER BY` in `OVER` works similarly to a query-level ORDER BY
+    /// ([`Self::bind_order_by_expr`]), but cannot use output-column names or numbers.
```

Comment:
As the implementation suggests, they are actually not similar 😂 But it is still okay to keep this function and possibly reuse it for `order_by` inside agg call.